engine/src/test/java/org/operaton/bpm/engine/test/api/mgmt/BatchQueryTest.java

```diff
@@ -128,13 +119,8 @@ void testBatchQueryById() {
   @Test
   void testBatchQueryByIdNull() {
     var batchQuery = managementService.createBatchQuery();
-    try {
-      batchQuery.batchId(null);
-      fail("exception expected");
-    }
-    catch (NullValueException e) {
-      assertThat(e.getMessage()).contains("Batch id is null");
-    }
+    assertThatThrownBy(() -> batchQuery.batchId(null)).isInstanceOf(NullValueException.class)
```

Comment:
nit: `.isInstanceOf` in new line, indent is 2 spaces
similar formatting issues at several places